docs/robots/jackal/tutorials_jackal.mdx

```diff
@@ -4,9 +4,1317 @@ sidebar_position: 3
+```
+
+When you run this command, you'll have access to `rosrun`, `roslaunch`, and all the other tools and packages installed on your system from Debian packages.
+
+However, sometimes you want to add additional system-specific environment variables, or perhaps packages built from source.
+For this reason, Clearpath platforms use a wrapper setup file, located in `/etc/ros`:
+
+```
+source /etc/ros/setup.bash
+```
+
+This is the setup file which gets sourced by Jackal's background launch job, and in the default configuration, it is also sourced on your login session.
+For this reason it can be considered the "global" setup file for Jackal's ROS installation.
+
+This file sets some environment variables and then sources a chosen ROS workspace, so it is one of your primary modification points for altering how Jackal launches.
+
+#### Launch Files
+
+The second major modification point is the `/etc/ros/noetic/ros.d` directory.
+This location contains the launch files associated with the `ros` background job.
+If you add launch files here, they will be launched with Jackal's startup.
+
+However, it's important to note that in the default configuration, any launch files you add may only reference ROS software installed in `/opt/ros/noetic`.
+If you want to launch something from workspace in the home directory, you must change `/etc/ros/setup.bash` to source that workspace's setup file rather than the one from `opt`.
+
+#### Adding URDF
+
+There are two possible approaches to augmenting Jackal's URDF.
+The first is that you may simply set the `Jackal_URDF_EXTRAS` environment variable in `/etc/ros/setup.bash`.
+By default, it points to an empty dummy file, but you can point it to a file of additional links and joints which you would like mixed into Jackal's URDF (via xacro) at runtime.
+
+The second, more sophisticated way to modify the URDF is to create a _new_ package for your own robot, and build your own URDF which wraps the one provided by [jackal_description](https://github.com/jackal/jackal/tree/noetic-devel/jackal_description).
+
+### Keeping Jackal Updated
+
+:::note
+
+If you are upgrading your Jackal from an older version of ROS, please refer to our upgrade instructions [here](https://clearpathrobotics.com/assets/guides/kinetic/kinetic-to-melodic/index.html) and [here](https://clearpathrobotics.com/assets/guides/melodic/melodic-to-noetic/index.html).
+
+:::
+
+#### Getting New Packages
+
+Jackal is always being improved, both its own software and the many community ROS packages upon which it depends!
+You can use the `apt` package management system to receive new versions all software running on the platform.
+
+Each Jackal leaves the factory already configured to pull packages from http://packages.ros.org as well as http://packages.clearpathrobotics.com.
+To update your package and download new package versions, simply run:
+
+```
+sudo apt-get update
+sudo apt-get dist-upgrade
+```
+
+#### MCU Firmware Update
```

Comment:
Should this be in Tutorials or in Maintenance?
MCU firmware seems to blue the line between using Noetic packages on the robot, and maintaining the robots functionality.

As a similar situation; we have Puma firmware updates in the Maintenance section of Husky.

Reply:
Moved all software update info to maintenance as discussed. Made the similar change for Dingo and Husky also.